src/ds3/ds3_mocks_test.go

```diff
@@ -37,78 +37,84 @@ type mockedNet struct {
     headers *http.Header
 }
 
-func (self *mockedNet) Expecting(verb networking.HttpVerb, path string, queryParams *url.Values, request *string) mockedClientWithExpectation {
-    self.verb = verb
-    self.path = path
-    self.queryParams = queryParams
-    self.request = request
-    return self
+func (mockedNet *mockedNet) Expecting(verb networking.HttpVerb, path string, queryParams *url.Values, request *string) mockedClientWithExpectation {
```

Comment:
should the *mockedNet type be capitalized?  I guess it looks like other "types" are not capitalized, just making sure we are consistent, since the ClientBuilder above is capitalized